layouts/in.vue

```diff
@@ -91,7 +96,10 @@ export default {
     ]),
   },
   mounted() {
-    if (checkIsMobileClient()) {
+    if (
+      checkIsMobileClient()
+      && ROUTES_ALLOW_LIST_FOR_MOBILE.includes(this.$route.name)
```

Comment:
I think should be `!ROUTES_ALLOW_LIST_FOR_MOBILE.includes(this.$route.name)` if we want that to be allow list